supporting_files/ArrayExpressSiteConfig.yml

```diff
@@ -78,6 +78,7 @@ VALIDATION_SCRIPT:
 # Location of ADF and Experiments checked in Atlas.pm
 ADF_CHECKED_LIST:        /nfs/production3/ma/home/atlas3-production/sw/configs/adfs_not_in_atlas.txt
 ATLAS_EXPT_CHECKED_LIST: /nfs/production3/ma/home/atlas3-production/sw/configs/expts_checked_for_atlas.txt
+SKIP_CHECKED_LIST_FILES:  true
```

Comment:
if this is not in place, does `$CONFIG->get_SKIP_CHECKED_LIST_FILES` evaluate to false or gives an error? I wonder since the source of this file in atlas in the cluster might not be this file. Or have you added this line to the file that atlas uses in the cluster?

Reply:
Maybe we can add a quick line in the bats test to see if this works?